compiler/luci/import/src/Importer.test.cpp

```diff
@@ -16,13 +16,289 @@
+  model.add_plan_entry(metadata_buffer_id, 3, {200});
+
+  flatbuffers::FlatBufferBuilder fbb;
+  auto model_offset = circle::Model::Pack(fbb, model.model.get(), nullptr);
+  circle::FinishModelBuffer(fbb, model_offset);
+
+  auto model_ptr = circle::GetModel(fbb.GetBufferPointer());
+  luci::Importer import;
+
+  auto luci_module = import.importModule(model_ptr);
+
+  auto main_graph = luci_module->graph();
+  for (int i = 0; i < main_graph->nodes()->size(); ++i)
+  {
+    auto node = loco::must_cast<luci::CircleNode *>(main_graph->nodes()->at(i));
+    switch (node->opcode())
+    {
+      case luci::CircleOpcode::CIRCLEINPUT:
+      {
+        ASSERT_TRUE(luci::has_execution_plan(node));
+        auto plan = luci::get_execution_plan(node);
+        ASSERT_EQ(plan.order_in_plan(), 1);
+        ASSERT_EQ(plan.offsets().size(), 1);
+        ASSERT_EQ(plan.offsets()[0], 100);
+        break;
+      }
+      case luci::CircleOpcode::CIRCLEOUTPUT:
+      {
+        ASSERT_TRUE(luci::has_execution_plan(node));
+        auto plan = luci::get_execution_plan(node);
+        ASSERT_EQ(plan.order_in_plan(), 3);
+        ASSERT_EQ(plan.offsets().size(), 1);
+        ASSERT_EQ(plan.offsets()[0], 200);
+        break;
+      }
+      case luci::CircleOpcode::RELU:
+      {
+        ASSERT_TRUE(luci::has_execution_plan(node));
+        auto plan = luci::get_execution_plan(node);
+        ASSERT_EQ(plan.order_in_plan(), 2);
+        ASSERT_EQ(plan.offsets().size(), 1);
+        ASSERT_EQ(plan.offsets()[0], 300);
+        break;
+      }
+    }
+  }
+}
+
+/**
+ * This test checks that model with incomplete execution plan is successfully imported
+ */
+TEST(TensorFlowLiteImporter, DISABLED_incomplete_plan_NEG)
+{
+  SimpleRELUModel model;
+  auto metadata_buffer_id = model.add_buffer();
+  model.add_plan_metadata(metadata_buffer_id);
+
+  model.add_plan_entry(metadata_buffer_id, 1, {100});
+
+  flatbuffers::FlatBufferBuilder fbb;
+  auto model_offset = circle::Model::Pack(fbb, model.model.get(), nullptr);
+  circle::FinishModelBuffer(fbb, model_offset);
+
+  auto model_ptr = circle::GetModel(fbb.GetBufferPointer());
+  luci::Importer import;
+
+  auto luci_module = import.importModule(model_ptr);
+
+  auto main_graph = luci_module->graph();
+  for (int i = 0; i < main_graph->nodes()->size(); ++i)
+  {
+    auto node = loco::must_cast<luci::CircleNode *>(main_graph->nodes()->at(i));
+    switch (node->opcode())
+    {
+      case luci::CircleOpcode::CIRCLEINPUT:
+      {
+        ASSERT_TRUE(luci::has_execution_plan(node));
+        auto plan = luci::get_execution_plan(node);
+        ASSERT_EQ(plan.order_in_plan(), 1);
+        ASSERT_EQ(plan.offsets().size(), 1);
+        ASSERT_EQ(plan.offsets()[0], 100);
+        break;
+      }
+      case luci::CircleOpcode::CIRCLEOUTPUT:
+      {
+        case luci::CircleOpcode::RELU:
+        {
+          ASSERT_FALSE(luci::has_execution_plan(node));
+          break;
+        }
+      }
```

Comment:
(optional) How about?

```suggestion
      case luci::CircleOpcode::RELU:
      {
        ASSERT_FALSE(luci::has_execution_plan(node));
        break;
      }
```

Reply:
I think this braces some redundant.

Reply:
yep, looks like I messed up with my keyboard and editor =)